common/schemamigration.go

```diff
@@ -35,7 +36,7 @@ func initSchema(schema string, name string) {
 		return
 	}
 
-	skip, err := checkSkipSchemaInit(schema, name)
+	skip, err := checkSkipSchemaInit(schema)

```

Comment:
While at it, I took the liberty of removing some commented-out code, unused parameters, and extraneous `return`s as suggested by the linter. These changes are purely stylistic in nature.